components/skus/browser/skus_sdk_context.cc

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/components/skus/browser/skus_sdk_context.h"
+
+#include <string>
+#include <utility>
+
+#include "base/threading/sequenced_task_runner_handle.h"
+#include "brave/components/skus/browser/brave-rewards-cxx/src/wrapper.h"
```

Comment:
`brave-rewards-cxx` doesn't follow chromium conventions. Presumably this shouldn't be `brave-rewards` anything, but when you change the name can you make sure it uses `_` instead of `-`?

Reply:
we should be following the conventions from speedreader which would be `rust/cxx`